packages/telemetry-node/src/telemetry.ts

```diff
@@ -0,0 +1,35 @@
+import LokiTransport from "winston-loki";
+import winston from "winston";
+
+const USER_NAME = "090360";
+const GRAFANA_BASIC_AUTH = `${USER_NAME}:${process.env.GRAFANA_TOKEN}`;
+
+const telemetryLogger = winston.createLogger({
+  level: "debug",
+  transports: [
+    new LokiTransport({
+      host: "https://logs-prod-006.grafana.net",
+      labels: { app: "playwright" },
+      json: true,
+      basicAuth: GRAFANA_BASIC_AUTH,
+      format: winston.format.json(),
+      replaceTimestamp: true,
+      onConnectionError: err => console.error(err),
+      gracefulShutdown: true,
+    }),
+  ],
+});
+
+type Tags = {
+  [key: string]: any;
+};
+
+export function telemetryDebug(message: string, tags?: Tags) {
+  if (process.env.REPLAY_TELEMETRY_DISABLED) {
+    return;
+  }
+
+  telemetryLogger.debug(message, tags);
+}
+
+export { telemetryLogger };
```

Comment:
I wonder if we should avoid exporting this directly, and force everything to go through an intermediate API instead (so we could enforce things like checking the opt-out `process.env.REPLAY_TELEMETRY_DISABLED`)

We'd also need to export a method that wrapped `telemetryLogger.close` but we might want to do that anyway to flush pending requests?

Reply:
Yeah, I thought I was only exporting `telemetryDebug` for this reason, so I'll fix this.

Speaking of `telemetryLogger.close`, I'm assuming we will need to figure out the best place to call that for each package. For the playwright package, I was considering putting it here:

https://github.com/replayio/replay-cli/blob/75ee1cb037de7e86f047ffe79dc84b1423c28aaa/packages/playwright/src/reporter.ts#L319

Reply:
Yeah, `onEnd` would likely be the correct place for this - maybe u could even put it in `onEnd` within the `test-utils`.

What does this method? is it mainly for flushing the events that may have not been sent out yet?

Reply:
I'm thinking ahead to if we pull Mixpanel into this package as well, as we have an explicit "wait a couple hundred ms to flush pending events before exciting" hook in the new CLI.

Reply:
I agree @bvaughn and @Andarist .

When we add Mixpanel to this package, I think we should merge both functionalities under the same function.

Reply:
> Yeah, `onEnd` would likely be the correct place for this - maybe u could even put it in `onEnd` within the `test-utils`.
> 
> What does this method? is it mainly for flushing the events that may have not been sent out yet?

Yes.